internal/common/objects/BaseBiker.go

```diff
@@ -241,3 +241,11 @@ func (bb *BaseBiker) SetGameState(gameState IGameState) {
 func (bb *BaseBiker) SetAllocationParams(allocationParams ResourceAllocationParams) {
 	bb.allocationParams = allocationParams
 }
+
+func (bb *BaseBiker) GetGameState() IGameState {
```

Comment:
These two i don't think will be necessary since when you inherit BaseBiker you can access the private members like megaBikeId, and gameState

Reply:
As discussed, Go does not allow this unfortunately so we're in agreement that this should be here. Gonna just merge the change!